utilities/command-string-factory.js

```diff
@@ -0,0 +1,96 @@
+// -----------------------------------------------------------------------------------------
+// #region Imports
+// -----------------------------------------------------------------------------------------
+
+const {
+    CollectionUtils,
+    StringUtils,
+} = require("andculturecode-javascript-core");
+const echo = require("../modules/echo");
+const shell = require("shelljs");
+
+// #endregion Imports
+
+// -----------------------------------------------------------------------------------------
+// #region Private Functions
+// -----------------------------------------------------------------------------------------
+
+const _buildString = (cmd = "", args = []) => {
+    let fullCommandString = cmd;
+
+    // Only append a space with the concatenated array of args if it contains a value
+    // Some commands do not require any additional arguments.
+    if (
+        CollectionUtils.hasValues(args) &&
+        StringUtils.hasValue(args.join(" "))
+    ) {
+        fullCommandString = `${fullCommandString} ${args.join(" ")}`;
+    }
+
+    return fullCommandString;
+};
+
+const _sanitizeArgs = (args = []) =>
```

Comment:
Unless I'm missing it, I don't think this is being used anywhere.

Reply:
This sanitization method is called after the validation step - see line 82 😛 Probably didn't need its own method, but I figured it might be nice to chunk out some of the extra logic in there similar to the `_validateOrExit` function